recipes/pyvips/meta.yaml

```diff
@@ -0,0 +1,61 @@
+# pyvips supports cffi ABI and API mode
+#
+# During setup.py, it will attempt to compile a generated wrapper against the
+# libvips headers -- this is why it needs pkgconfig and libvips in host.
+#
+# At runtime, it will first try to load the compiled wrapper, and if that
+# fails, will fall back to ABI mode, ie. directly opening the libvips shared
+# library and calling into it with cffi.
+
+# Skip win since libvips is not packaged for conda there. It is blocked on 
+# at least gdk-pixbuf.
+
+{% set name = "pyvips" %}
+{% set version = "2.1.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/libvips/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 4dcd1d6689f3faf46b340e70ee238bfd323b9757c5aff8d22a9e05201ebf2cc3
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - libvips
+    - pkgconfig         # [not win]
+  host:
+    - cffi >=1.0
+    - libvips
+    - pip
+    - pkgconfig         # [not win]
```

Comment:
This belongs in build:
```suggestion
```
Dependencies should (almost) never be duplicated between host and build.

Reply:
Hello @chrisburr, that's exactly the bit that took me ages to get right. You'll find it stops working if you remove that line.

pyvips is perhaps unusual in that it uses `cffi` API mode. At `bdist_build` time, it generates a C extension that implements the low level parts of the Python -> libvips binding, compiles it, and includes that `.so`/`.dll` with the byte-compiled .py files.

For this to work, it must have `pkgconfig`, so it can locate the libvips headers, libvips itself, and of course a C compiler. I didn't have to ask for the python-dev headers I guess they are always there.

I didn't try removing `pkgconfig` from `build`, that might be possible.

Reply:
Sorry I was thinking of `pkg-config` as opposed to the Python bindings (`pkgconfig`). You're correct, the Python bindings belong in `host`.